examples/cactus-example-supply-chain-frontend/package.json

```diff
@@ -47,6 +48,7 @@
     "@types/jasmine": "~3.5.0",
     "@types/jasminewd2": "~2.0.3",
     "@types/node": "^12.11.1",
+    "@types/ssh2": "0.5.46",
```

Comment:
@jordigiam Did this turn out to be really necessary or was that compilation error only happening prior to running `npm run build:dev` (which produces the builds for the browser platform as well)

Reply:
Done